MeshLib/MeshGenerators/LayeredMeshGenerator.cpp

```diff
@@ -56,7 +61,20 @@ MeshLib::Mesh* LayeredMeshGenerator::getMesh(std::string const& mesh_name) const
     if (_nodes.empty() || _elements.empty())
         return nullptr;
 
-    MeshLib::Mesh* result (new MeshLib::Mesh(mesh_name, _nodes, _elements));
+	MeshLib::Properties properties;
+	if (!_materials.empty())
```

Comment:
`_materials` being empty is covered by the next test of `_materials`' sizes being equal to `_elements`' sizes.